crate_universe/src/rendering/templates/module_bzl.j2

```diff
@@ -313,7 +314,7 @@ _BUILD_PROC_MACRO_DEPENDENCIES = {% set deps_type = "build-proc-macro" %}{% incl
 _BUILD_PROC_MACRO_ALIASES = {% set deps_type = "build-proc-macro" %}{% include "partials/module/aliases_map.j2" %}
 
 _CONDITIONS = {
-{%- for condition, triples in context.conditions %}
+{%- for condition, triples in platforms %}
```

Comment:
Can you update the `crates_vendor` examples so I can see a rendering of this?

Reply:
Done